views/default/reportabug.html

```diff
@@ -0,0 +1,44 @@
+{{ extend 'layout.html'}}
+
+<h1 style="text-align: center">Report a Bug</h1>
+
+<p>Sorry you are having trouble. I want to hear about problems you are having but before I send you off to github to report a bug please look over the following list of common problems and solutions:</p>
+<p>I can’t login, I can reset my password, but after that I can’t login again! This is definitely one of the most reported bugs. However in almost 100% of the cases so far, the problem is that you are using your email address as your username. When you registered you created a username, this is what you must use to login, not your email. IF you really are using the username you made and still having login problems then I want to hear from you.</p>
+<p>Very often, problems with activecode or exercises can be resolved by reloading your page a couple of times, or by clearing your cache.</p>
+<p>OK, now you are still running into some problem. Telling me that X does not work doesn’t really help. interactivepython.org gets over 30,000 pageviews a day from people around the world. So most of the time when someone says “X does not work” it does in fact work fine for most people. This does not make it your fault, it just means I need more information to try to make sense of what is going on. Fill out the following form to submit an error report on Github so that I can fix the problem.</p>
+
+<div class="col-md-7">
+    <form name="bugform" id="bugform" action="/{{=request.application}}/{{=request.controller}}/sendreport" method="post">
+        <div class="form-group row">
+        <div class="col-md-4">
+          <label for="bugtitle">Bug Title</label>
+          <input type="text" class='form-control' name="bugtitle" id="bugtitle" required/>
+        </div>
+      </div>
+        <p>Enter a short title that summarises the bug</p>
+        <div class="form-group row">
+        <div class="col-md-4">
+          <label for="coursename">Course Name</label>
+          <input type="text" class='form-control' placeholder="thinkcspy, pythonds, or other" name="coursename" id="coursename" required/>
+        </div>
+      </div>
+       <div class="form-group row">
+        <div class="col-md-4">
+          <label for="pagename">Page URL</label>
+          <input type="text" class='form-control' placeholder="Page URL of the bug" name="pagename" id="pagename" required/>
+        </div>
+      </div>
```

Comment:
I think you can autofill the values for course name and page url using information from the request object.  Especially since the course name is part of the url. I think it it will be in `referring page` or some other attribute of request.